packages/elastic-apm-synthtrace/src/lib/apm/apm_fields.ts

```diff
@@ -47,8 +52,7 @@ export type ApmFields = Fields &
     'host.name': string;
     'kubernetes.pod.uid': string;
     'metricset.name': string;
-    'observer.version': string;
-    'observer.version_major': number;
+    observer: Observer;
```

Comment:
any reason why these are no longer flattened fields? (i'd prefer to keep it consistent)

Reply:
I was initially sending all data to Elasticsearch flattened but hit an in issue with the ingest pipelines we ship with: 

https://github.com/elastic/apm-server/issues/7088

Later I realize we `dedot` before sending data originally so included that back in. Will revert observer on `ApmFields` to be flat :+1: